packages/react/src/hooks/participant-hooks.ts

```diff
@@ -14,38 +15,36 @@ import {
   mutedObserver,
   ParticipantMedia,
   observeParticipantMedia,
-  ParticipantFilter,
   participantPermissionObserver,
   connectedParticipantObserver,
+  UpdateOnlyOn,
+  toRoomEvents,
 } from '@livekit/components-core';
 import { useEnsureParticipant, useRoomContext } from '../context';
 import { useObservableState } from '../helper/useObservableState';
 
 export interface UseParticipantsOptions {
-  filter?: ParticipantFilter;
-  filterDependencies?: Array<unknown>;
+  updateOnlyOn?: UpdateOnlyOn;
```

Comment:
This type doesn't make it clear, that in most cases you would want to use the grouped room events

Reply:
it suggests passing in room events directly

Reply:
Agreed. Do you have a name in mind?